sdk/openai/openai/samples/v1-beta/javascript/README.md

```diff
@@ -48,25 +52,29 @@ npm install
 3. Run whichever samples you like (note that some samples may require additional setup, see the table above):
 
 ```bash
-node chatCompletions.js
+node bringYourOwnData.js
 ```
 
 Alternatively, run a single sample with the correct environment variables set (setting up the `.env` file is not required if you do this), for example (cross-platform):
 
 ```bash
-npx cross-env ENDPOINT="<endpoint>" AZURE_API_KEY="<azure api key>" node chatCompletions.js
+npx cross-env ENDPOINT="<endpoint>" AZURE_API_KEY="<azure api key>" AZURE_SEARCH_ENDPOINT="<azure search endpoint>" AZURE_SEARCH_KEY="<azure search key>" AZURE_SEARCH_INDEX="<azure search index>" node bringYourOwnData.js
```

Comment:
Do we specifically say here that it requires the `cross-env` npm module?

Reply:
`npx` is used and should install it automatically for you, no?

Reply:
Not necessarily, for example on Mac it doesn't: 

```bash
npx cross-env FOO="bar" dist/index.js
sh: cross-env: command not found
```

Reply:
Interesting, @witemple-msft could this be a gap in our sample READMEs?